Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -43,6 +43,7 @@ public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCas
                 case GT.MultiacademyTrust:
                 case GT.SingleacademyTrust:
                 case GT.SchoolSponsor:
+                case GT.SecureSingleAcademyTrust:
```

Comment:
This file has the following logic for SATs - does it need to be updated to also reference Secure SAT?

https://github.com/DFE-Digital/get-information-about-schools/blob/secure_16-19_groups/Web/Edubase.Web.UI/Areas/Groups/Views/Group/EditDetails.cshtml#L132-L143

```csharp
                    @if (Model.GroupType.OneOfThese(L.MultiacademyTrust, L.SingleacademyTrust, L.SchoolSponsor))
                    {
                        <div class="govuk-form-group @Html.ValidationCssClassFor(x => x.GroupId)">
                            <label for="GroupId" class="govuk-label">Group identifier (ID)</label>
                            @Html.ValidationMessageFor(x => x.GroupId, null, new { @class = "govuk-error-message" })
                            @Html.TextBoxFor(x => x.GroupId, new { @class = "govuk-input", maxlength = 120 })
                        </div>
                    }
                    else
                    {
                        @Html.HiddenFor(x => x.GroupId)
                    }
```